assets/pages/new-contributions-landing/components/ContributionBackground.jsx

```diff
@@ -3,15 +3,48 @@
 // ----- Imports ----- //
 
 import React from 'react';
-
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import SvgContributionsBgDesktop from 'components/svgs/contributionsBgDesktop';
+import GridPicture from 'components/gridPicture/gridPicture';
+
+type PropTypes = {|
+  countryGroupId: CountryGroupId,
+|};
 
 // ----- Render ----- //
 
-function NewContributionBackground() {
+function NewContributionBackground(props: PropTypes) {
+  if (props.countryGroupId !== 'UnitedStates') {
```

Comment:
could be slightly clearer to swap these round (`if (props.countryGroupId === 'UnitedStates')` first)